alert-policies/lighttpd/AverageNumberOfConnections.yml

```diff
@@ -0,0 +1,36 @@
+name: Average Number of Connections
+
+description: |+
+  This alert is triggered when the average number of connections are above 6000 for atleast 2 minutes.
+type: STATIC
+nrql:
+  query: "FROM lighttpdStatusSample2 SELECT latest(connectionsAvg)"
+
+# Function used to aggregate the NRQL query value(s) for comparison to the terms.threshold (Default: SINGLE_VALUE)
+valueFunction: SINGLE_VALUE
+
+# List of Critical and Warning thresholds for the condition
+terms:
+  - priority: CRITICAL
+    # Operator used to compare against the threshold.
+    operator: ABOVE
+    # Value that triggers a violation
+    threshold: 6000
+    # Time in seconds; 120 - 3600
+    thresholdDuration: 120
+    # How many data points must be in violation for the duration
+    thresholdOccurrences: ALL
+
+  - priority: WARNING
+    # Operator used to compare against the threshold.
```

Comment:
Why do some of these comments end in periods and not others. I prefer consistency :)